arpack-ng-config.cmake.in

```diff
@@ -20,6 +20,6 @@ set(includedir "@includedir@")
 # Create arpack targets.
 add_library(ARPACK::ARPACK INTERFACE IMPORTED)
 set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_INCLUDE_DIRECTORIES "${includedir}/arpack")
-set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack")
+set_target_properties(ARPACK::ARPACK PROPERTIES INTERFACE_LINK_LIBRARIES      "arpack@LIBSUFFIX@")
 add_library(PARPACK::PARPACK INTERFACE IMPORTED)
-set_target_properties(PARPACK::PARPACK PROPERTIES INTERFACE_LINK_LIBRARIES    "parpack")
+set_target_properties(PARPACK::PARPACK PROPERTIES INTERFACE_LINK_LIBRARIES    "parpack@LIBSUFFIX@")
```

Comment:
Did you test that? Is the install dir usable through `find_package`? Why not "${libdir}/parpack..."?
Does this imply to change install `parpack` targets in CMakeLists.txt?

No time yet to test this: *.pc and *.cmake are _not_ tested https://github.com/opencollab/arpack-ng/pull/259#issuecomment-603187398, changing them must be tested by hand